fastlane/Fastfile

```diff
@@ -0,0 +1,20 @@
+default_platform(:ios)
+
+platform :ios do
+  lane :fetch_install_certs do
+    cert(
+      keychain_path: ENV['KEYCHAIN_PATH'],
+      generate_apple_certs: "false"
+    )
+  end
+
+  lane :fetch_install_provisioning do
+    sigh(
+      app_identifier: ENV['APP_BUNDLE_ID']
+    )
+    sigh(
+      app_identifier: ENV['APP_BUNDLE_ID'],
+      adhoc: "true"
+    )
```

Comment:
Based on looking stuff up this looks like we're getting an App Store provisioning profile and an ad hoc provisioning profile. Why are we doing both? Because we're building several different types of binaries?

Reply:
I added the App Store one too because I figure at some point in the future we'll need that too if we want to push to TestFlight. But I can comment that out for now.

Reply:
I don't mind, was just curious.

Reply:
I take that back, we do need it because the Xcode project is configured to use that provisioning profile during archiving by default, and we swap it only it during exporting.